libhttpserver/server_test.go

```diff
@@ -0,0 +1,75 @@
+// Copyright 2018 Keybase Inc. All rights reserved.
+// Use of this source code is governed by a BSD
+// license that can be found in the LICENSE file.
+
+package libhttpserver
+
+import (
+	"fmt"
+	"net/http"
+	"os"
+	"testing"
+
+	"github.com/keybase/client/go/libkb"
+	"github.com/keybase/kbfs/ioutil"
+	"github.com/keybase/kbfs/libkbfs"
+	"github.com/stretchr/testify/require"
+)
+
+func makeTestKBFSConfig(t *testing.T) (
+	kbfsConfig libkbfs.Config, shutdown func()) {
+	ctx := libkbfs.BackgroundContextWithCancellationDelayer()
+	cfg := libkbfs.MakeTestConfigOrBustLoggedInWithMode(
+		t, 0, libkbfs.InitSingleOp, "alice", "bob")
+
+	tempdir, err := ioutil.TempDir(os.TempDir(), "journal_server")
```

Comment:
Nit: if this function hits an error before it returns, `tempdir` will never be cleaned up.  Usually I'd have this function return an error, and then use a `defer` that does clean up if `err != nil`.